### ISSUE TITLE
Right side menu on Options page is clipped

### ISSUE THREAD

**Issue by imiric** [k6io/docs#216]
A minor UI issue: on Chromium 87 with 1920x1080 resolution and 150% page zoom the right side menu on the [Options page](https://k6.io/docs/using-k6/options) overflows the window boundary and gets clipped. It's also annoying because horizontal scrolling is disabled so the content is invisible unless I zoom out or enlarge the window.

Here's what it looks like for me:
![2021-02-10-164507_1920x1080_scrot](https://user-images.githubusercontent.com/1009277/107533881-ac629700-6bbf-11eb-81c8-38c3328efb93.png)

It looks fine with 125% zoom and below, and with >150% zoom since the layout switches to mobile. It also looks fine in Firefox 84 at 150% zoom as the main column is narrower than on Chromium, which is how I think it's meant to look. So this seems to only affect Chrome/WebKit (it's also a problem on Luakit and I reckon any other WebKit browser).

One could argue that I shouldn't be using zoom, but I'd argue back that the UI should be responsive to handle all zoom levels and resolutions gracefully. :)

For this page specifically I'd suggest just dropping the right side menu entirely, since it only has 2 items so it's not that useful, and the page is loaded with text so it could benefit from a wider main content column.

**Comment by MattDodsonEnglish** [comment]
I don't think this a problem anymore. At least, I couldn't reproduce.  And the page is changed.

@imiric can you confirm that the right-side menu works on all zooms for you in Chrome? Then we can close.
https://k6.io/docs/using-k6/k6-options/reference/
